core/src/main/java/com/graphhopper/routing/ev/DefaultImportRegistry.java

```diff
@@ -51,11 +54,28 @@ else if (RoadEnvironment.KEY.equals(name))
                     (lookup, props) -> new OSMRoadEnvironmentParser(
                             lookup.getEnumEncodedValue(RoadEnvironment.KEY, RoadEnvironment.class))
             );
+        else if (FootRoadAccess.KEY.equals(name))
+            return ImportUnit.create(name, props -> FootRoadAccess.create(),
+                    (lookup, props) -> new OSMRoadAccessParser<>(
+                            lookup.getEnumEncodedValue(FootRoadAccess.KEY, FootRoadAccess.class),
+                            OSMRoadAccessParser.toOSMRestrictions(TransportationMode.FOOT),
```

Comment:
Ideally we should be able to pass the TransportationMode directly as explained in the CountryRule related comment.

Reply:
An OSMRoadAccessParser is bound to a particular TransportationMode. Currently this is done indirectly as we pass the resolved restrictions and a country hook which is also TransportationMode dependent (or at least it should/will be).

If we pass the TransportationMode, we can do the resolution inside the parser and call the countryrule directly without a hook.